### ISSUE TITLE
How to toggle if cursor not in keyword

### ISSUE THREAD

**Issue by jkeks** [gordio/ToggleWords#19]
if i has a line:

$renameAsFirstLine = false; // long comment 256 byte
and cursor in a start of line, I want to toggle it too. 
I think first find keyword must be toggled !

yep ?

**Comment by gordio** [comment]
Good idea, if there’s no ”toggle keywords“ just try with whole line. 👍🏻
Can’t implement it right now, but definitely need to

**Comment by gordio** [comment]
Sorry, still don't have time.

**Comment by gordio** [comment]
Upd: Do some tests and found issue on lines like `disabled=true`. So will implement it optional
